src/main/java/io/anserini/util/DumpAnalyzedQueries.java

```diff
@@ -63,6 +94,91 @@ public static void main(String[] argv) throws IOException {
       return;
     }
 
+    // Read the analyzer from the command line, default is en 
+    if (args.language.equals("ar")) {
```

Comment:
Can you use [AnalyzerMap](https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/analysis/AnalyzerMap.java) here instead?

Reply:
What about `sw` or `te` though? Perhaps we should add a "whitespace" language, or just allow a default to be set? Otherwise using the AnalyzerMap wont allow all languages to be dealt with correctly I guess.

I lifted this from the SimpleSearcher just in case this helps with deciding how to deal with whitespace: https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/search/SimpleSearcher.java#L196

Reply:
You could use a `getAnalyzer` method similar to [SimpleIndexer.getAnalyzer](https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/SimpleIndexer.java#L127-L150)

Reply:
Hey @theyorubayesian - how about this?